news_flash/geocode_extraction.py

```diff
@@ -0,0 +1,12 @@
+import googlemaps
+
+
+def geocode_extract(location, maps_key):
+    gmaps = googlemaps.Client(key=maps_key)
+    geocode_result = gmaps.geocode(location, language="iw", region="il")
+    lat = geocode_result[0]["geometry"]["location"]["lat"]
+    lng = geocode_result[0]["geometry"]["location"]["lng"]
+    if 29.479700 <= lat <= 33.332805 and 34.267387 <= lng <= 35.896244:
```

Comment:
why do you check that? are those Israel's גבולות גזרה? I'm pretty sure google maps api returns the country as well, so you can check that instead if you want to make sure. (and if not I think we need a comment here or constants that explain this)

Reply:
Found something and replaced this check with that. Thanks.